diff -Nur iplike-pgsql81-1.0.5.old/build-tree/iplike-1.0.5-1/install_iplike.sh.in iplike-pgsql81-1.0.5/build-tree/iplike-1.0.5-1/install_iplike.sh.in
--- iplike-1.0.5-1/install_iplike.sh.in  2007-06-19 12:53:22.000000000 -0400
+++ iplike-1.0.5-1/install_iplike.sh.in      2007-06-23 15:50:54.000000000 -0400
@@ -52,7 +52,7 @@
        do_error "unable to locate iplike.la"
 fi
 
-source "$libdir/iplike.la"
+. "$libdir/iplike.la"
 
 if [ ! -f "$libdir/$dlname" ]; then
        do_error "$libdir/$dlname does not exist"
